cmd/algorand-indexer/daemon.go

```diff
@@ -34,6 +34,8 @@ var (
 	tokenString      string
 	writeTimeout     time.Duration
 	readTimeout      time.Duration
+	maxConn          int32
```

Comment:
should these be uint32 to disallow negative values?

Reply:
I chose int32 because the postgres config units are int32...Should I change them here and cast them when creating the Postgres config?

Reply:
Unless there's a reason to allow negative values, I think it would be better to use uint32 here and let the flag parser enforce the negative bounds checking for us.

Sometimes interfaces use uint32 to avoid casts when using other built-ins... for example:
```
if len(connections) >= maxConn {
    ...
}
```